src/lib.rs

```diff
@@ -142,6 +142,23 @@ impl<T> Vec<T> {
         self.raw.push(value)
     }
 
+    /// Appends the element returned from the closure `f` to the back of the vector
+    /// at the index supplied to the closure.
+    ///
```

Comment:
Would be good to add a note about the return value here.

```suggestion
    /// at the index supplied to the closure.
    ///
    /// Returns the index that the element was inserted into.
```

Reply:
Ugh I'm sorry, I just now saw this. GitHub isn't notifying me on replies/reviews for some reason.


I'll make the change as soon as I'm in front of my machine.